src/archivematicaCommon/lib/databaseFunctions.py

```diff
@@ -219,10 +221,14 @@ def logTaskCompletedSQL(task):
     task = Task.objects.get(taskuuid=taskUUID)
     task.endtime = getUTCDate()
     task.exitcode = exitCode
-    task.stdout = stdOut
-    task.stderror = stdError
+    # ``strToUnicode`` here prevents the MCP server from crashing when, e.g.,
+    # stderr contains Latin-1-encoded chars such as \xa9, i.e., the copyright
+    # symbol, cf. #9967.
+    task.stdout = strToUnicode(stdOut, True)
+    task.stderror = strToUnicode(stdError, True)
```

Comment:
Nitpick, but for clarity `strToUnicode(stdError, obstinate=True)` would be better.